cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -162,6 +167,11 @@ func VerifyBlobCmd(ctx context.Context, ko sign.KeyOpts, certRef, certEmail, cer
 		}
 	}
 
+	// Use the DSSE verifier if the payload is a DSSE with the In-Toto format.
+	if isIntotoDSSE, _ := isIntotoDSSE(blobBytes); isIntotoDSSE {
```

Comment:
let's either check the error here and return it, or change this function to only return a `bool` if the error should be ignored :)

Reply:
Copy that! Addressed ✅